src/DebugEngineHost.VSCode/HostMarshal.cs

```diff
@@ -110,7 +111,7 @@ public static IDebugFunctionPosition2 GetDebugFunctionPositionForIntPtr(IntPtr f
         /// <returns></returns>
         public static string GetDataBreakpointStringForIntPtr(IntPtr stringId)
         {
-            throw new NotImplementedException();
+            return Marshal.PtrToStringBSTR(stringId);
```

Comment:
>PtrToStringBSTR [](http://example.com/codeflow?start=27&length=15)

I am not sure this API will actually work on Linux/Mac.

Reply:
I tested it on my laptop which already has Debian 11 installed, seems to work fine.